.github/workflows/reusable_package_mage.yaml

```diff
@@ -126,8 +126,7 @@ jobs:
           if [[ "${{ env.MAGE_BUILD_TARGET }}" == 'dev' ]]; then
             ARTIFACT_NAME="${ARTIFACT_NAME}-dev"
             IMAGE_TAG="${IMAGE_TAG}-dev"
-          fi
-          if [[ "${{ env.MAGE_BUILD_TYPE }}" == 'RelWithDebInfo' ]]; then
+          elif [[ "${{ env.MAGE_BUILD_TYPE }}" == 'RelWithDebInfo' ]]; then
```

Comment:
Don't mix different tests within the same if/elseif/else structure. MAGE_BUILD_TARGET and MAGE_BUILD_TYPE look separate and distinct.

Reply:
Good catch. I fixed it.